netlify.toml

```diff
@@ -1,5 +1,5 @@
 [[redirects]]
   from = "/library"
-  to = "/library/get-started"
+  to = "/library/api-reference"
```

Comment:
@sfc-gh-jgarcia Since I didn't edit the workaround removing the `<Link>` component in [navItem.js](https://github.com/streamlit/docs/blob/2722637ff867ef8dd350782a558d54a3d3b4400f/components/navigation/navItem.js#L83), I thought editing `netlify.toml` should be sufficient. However, the preview is still redirecting /library to Get started even though I edited `netlify.toml` to redirect it to the API reference. Any ideas? Am I missing something?

Reply:
Actually, more weirdness. I was looking at the preview on my phone and the redirect works in Chrome on Android. More checking and it also works in Firefox on macOS Sonoma. It works in Chrome and Firefox on Windows 10 as well. It's just not working in Chrome on macOS Sonoma. (I updated Chrome to double-check, too.)

Reply:
Ok. It works in an incognito session, so it must be something cached. I'll presume this is a non-issue.